client/my-sites/purchases/subscriptions/site-level-purchase-links.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import { CompactCard } from '@automattic/components';
+
+export default function SiteLevelPurchaseLinks() {
+	const translate = useTranslate();
+	// TODO: Fix these links
+	return (
+		<>
+			<CompactCard href="/me/purchases">{ translate( 'View all subscriptions for' ) }</CompactCard>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **8** times:
`translate( 'View all subscriptions' )` *ES Score: 6*